Dockerfile

```diff
@@ -9,15 +9,17 @@
 #
 ###
 
-FROM alpine:3.2
+FROM alpine:latest
 MAINTAINER Kingsquare <docker@kingsquare.nl>
+MAINTAINER bubak4 <mar@centrum.cz>
```

Comment:
no problem, I just copied the line with my name as I thought you do it that way, I do not use it either, will remove